src/main/java/cn/nukkit/block/BlockPumpkin.java

```diff
@@ -1,20 +1,30 @@
 package cn.nukkit.block;
 
 import cn.nukkit.Player;
+import cn.nukkit.block.BlockCarvedPumpkin;
```

Comment:
This import is not being used (cn.nukkit.block.BlockCarvedPumpkin). Could you remove it?